### DISCUSSION TITLE
can we have a small commercial website on vercel free hosting

### DISCUSSION THREAD

**Discussion by ashley12-webdevelopment** [vercel/community#1411] [Help] [answered]
### Summary

Hi,
I would like to get some clarification on the following please. can we have commercial website for a small business which accept payments on the vercel free plan?

**Comment by swarnava** [comment, marked as answer]
Hi, This is prohibited in Hobby / Personal plan. Personal / Free account is meant for testing purpose and shouldn't use for Production usecase. Read more about Fair Usage Policy here: https://vercel.com/docs/concepts/limits/fair-use-policy

**Reply by ashley12-webdevelopment** [reply]
understood. Thank you very much for your feedback. Appreciate it
